src/css/sponsors.css

```diff
@@ -0,0 +1,27 @@
+.sponsor {
+  @apply px-4 py-2 m-2;
+}
+
+.contact-section__social-icon {
+  @apply bg-c600 opacity-75  w-10 h-10 rounded-full flex justify-center items-center m-1;
```

Comment:
add cursor pointer in social media icons

Reply:
added automatically when changed to links